catroid/src/test/java/org/catrobat/catroid/test/robolectric/bricks/BrickSingleFormulaFieldTest.java

```diff
@@ -151,139 +150,137 @@
-
-				{JumpingSumoMoveForwardBrick.class.getSimpleName() + " duration", new JumpingSumoMoveForwardBrick(), R.id.brick_jumping_sumo_move_forward_edit_text_second},
-				{JumpingSumoMoveForwardBrick.class.getSimpleName() + " power", new JumpingSumoMoveForwardBrick(), R.id.brick_jumping_sumo_move_forward_edit_text_power},
-				{JumpingSumoMoveBackwardBrick.class.getSimpleName() + " duration", new JumpingSumoMoveBackwardBrick(), R.id.brick_jumping_sumo_move_backward_edit_text_second},
-				{JumpingSumoMoveBackwardBrick.class.getSimpleName() + " power", new JumpingSumoMoveBackwardBrick(), R.id.brick_jumping_sumo_move_backward_edit_text_power},
-				{JumpingSumoSoundBrick.class.getSimpleName(), new JumpingSumoSoundBrick(), R.id.brick_jumping_sumo_sound_edit_text},
-				{JumpingSumoRotateLeftBrick.class.getSimpleName(), new JumpingSumoRotateLeftBrick(), R.id.brick_jumping_sumo_change_left_variable_edit_text},
-				{JumpingSumoRotateRightBrick.class.getSimpleName(), new JumpingSumoRotateRightBrick(), R.id.brick_jumping_sumo_change_right_variable_edit_text},
-
-				//Phiro bricks
-				{PhiroMotorMoveBackwardBrick.class.getSimpleName(), new PhiroMotorMoveBackwardBrick(), R.id.brick_phiro_motor_backward_action_speed_edit_text},
-				{PhiroMotorMoveForwardBrick.class.getSimpleName(), new PhiroMotorMoveForwardBrick(), R.id.brick_phiro_motor_forward_action_speed_edit_text},
-				{PhiroPlayToneBrick.class.getSimpleName(), new PhiroPlayToneBrick(), R.id.brick_phiro_play_tone_duration_edit_text},
-
-				//Arduino bricks
-				{ArduinoSendDigitalValueBrick.class.getSimpleName() + " pin", new ArduinoSendDigitalValueBrick(), R.id.brick_arduino_set_digital_pin_edit_text},
-				{ArduinoSendDigitalValueBrick.class.getSimpleName() + " value", new ArduinoSendDigitalValueBrick(), R.id.brick_arduino_set_digital_value_edit_text},
-				{ArduinoSendPWMValueBrick.class.getSimpleName() + " pin", new ArduinoSendPWMValueBrick(), R.id.brick_arduino_set_analog_pin_edit_text},
-				{ArduinoSendPWMValueBrick.class.getSimpleName() + " value", new ArduinoSendPWMValueBrick(), R.id.brick_arduino_set_analog_value_edit_text},
-
-				//Raspberry bricks
-				{RaspiSendDigitalValueBrick.class.getSimpleName() + " pin", new RaspiSendDigitalValueBrick(), R.id.brick_raspi_set_digital_pin_edit_text},
-				{RaspiSendDigitalValueBrick.class.getSimpleName() + " value", new RaspiSendDigitalValueBrick(), R.id.brick_raspi_set_digital_value_edit_text},
-				{RaspiPwmBrick.class.getSimpleName() + " pin", new RaspiPwmBrick(), R.id.brick_raspi_pwm_pin_edit_text},
-				{RaspiPwmBrick.class.getSimpleName() + " frequency", new RaspiPwmBrick(), R.id.brick_raspi_pwm_frequency_edit_text},
-				{RaspiPwmBrick.class.getSimpleName() + " percentage", new RaspiPwmBrick(), R.id.brick_raspi_pwm_percentage_edit_text},
-				{RaspiIfLogicBeginBrick.class.getSimpleName(), new RaspiIfLogicBeginBrick(), R.id.brick_if_begin_edit_text},
-
```

Comment:
I see no changes from line 154 until this line. please avoid reformating the whole file.
also please revert it and push only your changes.